.evergreen/run-tests.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+set -o errexit
```

Comment:
IIUC the drivers matrix testing project will eventually run this ([per these instructions](https://github.com/mongodb/drivers-matrix-testing#developer-guide)) after we create a branch for 1.6 and add it to the matrix to test: https://github.com/mongodb/drivers-matrix-testing/blob/main/matrix.yml#L117

Is it possible to run a patch build against drivers-matrix-testing against a test branch to verify this script works?

Reply:
Yeah, I think I can run a patch off this branch (i need to push changes first though)

Reply:
not done yet, but ubuntu and windows successfully ran tests: https://evergreen.mongodb.com/version/60ae8aab57e85a681fb60d2d

Reply:
Oh nice, thank you for running a patch. That makes sense. Azure/GCP CSFLE support and Versioned API variables are not defined in the drivers-matrix-testing [config.yml](https://github.com/mongodb/drivers-matrix-testing/blob/main/.evergreen/config.yml#L63). I think we (and all drivers) will need to address that when we add our 5.0-compat versions to the matrix project.